index.js

```diff
@@ -149,9 +147,18 @@ function fastifyView (fastify, opts, next) {
     return viewExt ? `.${viewExt}` : (includeViewExtension ? `.${extension}` : filextension)
   }
 
+  function isPathExcludedMinification (currentPath, pathsToExclude) {
+    return pathsToExclude.includes(currentPath)
+  }
+
+  function useHtmlMinification (globalOpts, requestedPath) {
+    return globalOptions.useHtmlMinifier &&
+      (typeof globalOptions.useHtmlMinifier.minify === 'function') &&
+      !isPathExcludedMinification(requestedPath, globalOptions.pathsToExcludeHtmlMinifier || [])
```

Comment:
Creating empty array stress the garbage collector more than a falsy check on the `globalOptions.pathsToExcludeHtmlMinifier` - in this function that is an hotpath

Reply:
Thanks for feedback, I removed empty array initialization and added this check to avoid Exceptions
`(pathsToExclude && Array.isArray(pathsToExclude))`